### PR TITLE
Added chain configuration and p2p rates/totals to dash display

### PR DESCRIPTION
**Pull request by ManushB**
Added chain name, id(morden or mainnet), chain id and p2p rx/tx rates/totals to the dash display.
I have attached screenshot for the same below.

screen shot 2018-02-05 at 11 49 38 pm

Signed-off-by: Manush <manush@lavellenetworks.com>

### CODE REVIEW

File: cmd/geth/log_display_dash.go [hunk 1]
```diff
@@ -85,13 +86,19 @@ var (
...
+func tuiDrawDash(e *eth.Ethereum, ctx *cli.Context) {
 	if currentMode == lsModeImport || currentMode == lsModeDiscover {
 		syncheightGauge.Label = ""
 	}
 	if e != nil && e.IsListening() {
 		cb := e.BlockChain().GetBlockByNumber(currentBlockNumber)
-		syncheightGauge.BorderLabel = fmt.Sprintf("%s | local_head ◼ n=%d ⬡=%s txs=%d time=%v ago", currentMode, currentBlockNumber, cb.Hash().Hex()[:10] + "…", cb.Transactions().Len(), time.Since(time.Unix(cb.Time().Int64(), 0)).Round(time.Second))
+		cid := e.ChainConfig().GetChainID()
+		cnet := e.NetVersion()
+		// Is there any other way retrieve chain name(mainnet or morden)?
```
Comment (whilei) [inline cmd/geth/log_display_dash.go]: Could use `cacheChainIdentity`, since it will have been set during node configuration.

**Comment by whilei** [comment]
screen shot 2018-02-07 at 16 18 23

I'm also noticing that the "metadata" text is getting cut off... I guess we could either use termui's "flex" layout, or move the data from being a border label to it's own text box. What do you think?

**Comment by ManushB** [comment]
screen shot 2018-02-08 at 10 48 07 am

Maybe we can have a header line which displays the config data. I can separate it into multiple rows to give further clarity.

**Comment by ManushB** [comment]
screen shot 2018-02-13 at 1 38 30 pm

Sorry for the delay, was caught up in few things. Does this look fine?

**Comment by whilei** [comment]
Looks good to me!

**Comment by ManushB** [comment]
Dash display with the latest changes.
screen shot 2018-02-14 at 12 24 33 am

**Comment by ManushB** [comment]
Hi @whilei , Do you want to add any other changes?

**Comment by whilei** [comment]
LGTM. If you'll just rebase on master then we can queue for merging 👍

**Comment by whilei** [comment]
Thanks @ManushB! Great to have your contribution. If you'd like to tackle another TODO around the dash, maybe we could implement TermUI's "flex" grid system, to escape hardcoded height and width? Or have you got any other ideas in mind?

**Comment by ManushB** [comment]
Thanks, @whilei for the help. I would like to work on other TODO items. Yes, we can use TermUI's 'flex' grid system for the same.

**Pull request merged**